apis/Google.Cloud.Shell.V1/Google.Cloud.Shell.V1/CloudShellServiceClient.g.cs

```diff
@@ -287,7 +293,19 @@ public abstract partial class CloudShellServiceClient
             "https://www.googleapis.com/auth/cloud-platform",
         });
 
-        internal static gaxgrpc::ChannelPool ChannelPool { get; } = new gaxgrpc::ChannelPool(DefaultScopes);
+        internal static gaxgrpc::ChannelPool ChannelPool { get; } = new gaxgrpc::ChannelPool(DefaultScopes, UseJwtAccessWithScopes);
+
+        internal static bool UseJwtAccessWithScopes
```

Comment:
Maybe do this a Lazy, since we use it for ChannelPool initialization once, and then several times on the ClientBuilder, we don't want the interceptor to change its mind about using JWTs.

Reply:
It never would, unless someone wrote a really odd partial method. (This is all internal or private.)